.github/workflows/zz_generated.create_release.yaml

```diff
@@ -1,12 +1,13 @@
 # DO NOT EDIT. Generated with:
 #
-#    devctl@4.1.0
+#    devctl@4.2.0
 #
 name: Create Release
 on:
   push:
     branches:
       - 'legacy'
+	  - 'main'
```

Comment:
@kubasobon I saw you merged `devctl` last. Just a shot in the dark. This here does not look right.

Reply:
https://gigantic.slack.com/archives/C3C7ZQXC1/p1612196571027400
It's been resolved and the PR should now be updated with correct data.
Sorry for the hassle :heart: